analyzers/src/SonarAnalyzer.CSharp/RspecStrings.resx

```diff
@@ -12844,6 +12844,39 @@
   <data name="S6354_Type" xml:space="preserve">
     <value>CODE_SMELL</value>
   </data>
+  <data name="S6421_Category" xml:space="preserve">
+    <value>Major Code Smell</value>
+  </data>
+  <data name="S6421_Description" xml:space="preserve">
+    <value>The top-most level of an Azure Function code should include a try/catch block to capture and log all errors so you can monitor the health of the application effectively. In case a retry policy has been defined for your Azure Function, you should rethrow any errors that should result in a retry.</value>
+  </data>
+  <data name="S6421_IsActivatedByDefault" xml:space="preserve">
+    <value>False</value>
+  </data>
+  <data name="S6421_Remediation" xml:space="preserve">
+    <value>Constant/Issue</value>
+  </data>
+  <data name="S6421_RemediationCost" xml:space="preserve">
+    <value>1min</value>
+  </data>
+  <data name="S6421_Scope" xml:space="preserve">
+    <value>All</value>
```

Comment:
This should be changed to `Main` and updated in the RSPEC - waiting for an answer on slack